runner/wormholes.rb

```diff
@@ -14,8 +12,8 @@
     s = System.where.not(security_status: :wormhole).where(security_status: :low).order(Arel.sql("RANDOM()")).first
 
     # Jumpgates
-    a = Location.find_or_create_by(name: sys.name, system: s, location_type: 5, hidden: true)
-    b = Location.find_or_create_by(name: s.name, system: sys, location_type: 5, hidden: true)
+    a = Location.where(name: sys.name, system: s, location_type: 5, hidden: true).first_or_create
```

Comment:
Why that change?

Reply:
OCD probably...  There was a time when `find_or_create_by` had some issues... I've done some [research](https://stackoverflow.com/questions/42178674/first-or-create-vs-find-or-create-by) and feel that you are correct -- I'll change these back.

Reply:
put `find_or_create_by` back